downloads.md

```diff
@@ -159,3 +159,25 @@ Installing NeoMutt is as simple as:
 ```
 xbps-install neomutt
 ```
+
+## NixOS
+
+Packagers:
+* [Charles Strahan](https://github.com/cstrahan),
+* [Christian Lask](https://github.com/hiberno),
+* [Rahul Gopinath](https://github.com/vrthra) and
+* [Erik Rybakken](https://github.com/erikryb)
+
+To install NeoMutt:
+
+add `pkgs.neomutt` into `configuration.nix` like that:
+
+```
+environment.systemPackages = [ pkgs.neomutt ];
```

Comment:
There are a few more options to install such as `nix-env`.  You may want to add a link to http://nixos.org/nix/manual/#chap-package-management . Otherwise it looks good.